app/routes/chatbot.js

```diff
@@ -271,8 +290,20 @@ router.post('/', (req, res) => {
     })
     .catch(err => {
       if (err.message === 'broadcast declined') {
-        const msg = campaignBot.renderMessage(scope, 'signup_broadcast_declined');
-        scope.user.postMobileCommonsProfileUpdate(agentViewOip, msg);
+        const msg = scope.broadcast.messages.prompt_declined;
+        if (!msg) {
+          const logMsg = 'undefined broadcast.messages.prompt_declined';
+          logger.error(logMsg);
+          stathat(`error: ${logMsg}`);
+
+          // Don't send an error code to Mobile Commons, which would trigger error message to User.
+          return helpers.sendResponse(res, 200, logMsg);
+        }
+
+        // Log the no response:
+        app.locals.db.bot_requests.log(req, 'broadcast', null, 'prompt_declined', msg);
```

Comment:
Seems slightly hacky to pass `broadcast` as `bot_type` and `null` as the `bot_id` -- but we're defining the `prompt_declined` copy on the `broadcast` model. We already are storing the `broadcast_id` within in the `body` object in the model, which is how we currently query the collection to determine YES responses.

<img width="858" alt="screen shot 2017-01-26 at 8 22 04 am" src="https://cloud.githubusercontent.com/assets/1236811/22339946/94b7fdf8-e3a0-11e6-9bb4-8b1e475912e7.png">